### ISSUE TITLE
version 2.0.0 search does nothing in Firefox

### ISSUE THREAD

**Issue by laniakea64** [brandon1024/find#255]
## Issue Description

Since version 2.0.0, trying to search for something doesn't do anything in Firefox.

This message in the Browser Console is related -

```
TypeError: self.browser is undefined app.js:25:9 
```

If I change this line
https://github.com/brandon1024/find/blob/1b2e6019e97124257a2f0371c8cd19d52c17a80b/app.js#L21
to
```
return typeof chrome === 'undefined' ? browser : chrome;
```
then everything works as expected.

## Steps to Reproduce

Firefox 65.0.1
new profile

1) go to https://github.com/brandon1024/find/

2) open find+'s popup, try to search for `find`

## Tasks
- [ ] t1
- [ ] t2
- [ ] t3

**Comment by brandon1024** [comment]
@laniakea64 Thank you for logging this issue. I will look into this right away.

**Comment by brandon1024** [comment]
@laniakea64 This issue was addressed, and a new release v2.0.1 was submitted. Again, thanks for bringing this to my attention.

**Comment by laniakea64** [comment]
You're welcome and I confirm that version 2.0.1 fixes this issue.  Thanks for the fast fix! :smiley:
